cli/cmd/cluster.go

```diff
@@ -169,8 +169,8 @@ func newGatewaysCommand() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVar(&opts.clusterName, "cluster-name", "remote", "the name of the remote cluster")
-	cmd.Flags().StringVar(&opts.gatewayNamespace, "gateway-namespace", defaultMulticlusterNamespace, "the namespace in which the gateway resides on the remote cluster")
+	cmd.Flags().StringVar(&opts.clusterName, "cluster-name", "", "the name of the remote cluster")
```

Comment:
just to clarify, leaving these blank means that the default is to look at all remote clusters?  and all gateway namespaces?

Reply:
Yes, this fixes the problem you originally mentioned where gateways were not showing up